protostar/commands/test/test_cases.py

```diff
@@ -7,27 +8,53 @@
 )
 from protostar.commands.test.test_environment_exceptions import ReportedException
 from protostar.protostar_exception import UNEXPECTED_PROTOSTAR_ERROR_MSG
-from protostar.utils.log_color_provider import log_color_provider
+from protostar.utils.log_color_provider import log_color_provider, SupportedColorName
+
+
+def _get_formatted_stdout(
```

Comment:
nitpick: could you please move these underscored functions to bottom of the file, so that the meat of this module stays on top? :)